app/serializers/score_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class ScoreSerializer < ActiveModel::Serializer
-  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :created_at, :updated_at
+  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :position, :created_at, :updated_at
```

Comment:
@tiagonbotelho Instead I added a position method which returns the score position for a given text ;)